docs/root/api/client_features.rst

```diff
@@ -23,3 +23,9 @@ Currently Defined Client Features
 - **envoy.lrs.supports_send_all_clusters**: This feature indicates that the client supports
   the *envoy_api_field_service.load_stats.v2.LoadStatsResponse.send_all_clusters*
   field in the LRS response.
+- **envoy.config.supports-resource-in-sotw-response**: This feature indicates that xDS client
```

Comment:
@markdroth I wonder if what we want to do to make context parameters sane here is to declare these features mandatory at some future minor version. Unless you have a strong preference to make this orthogonal for gRPC.

Reply:
I do think it's likely that there will be clients that won't support this feature, so it seems likely to want to remain an optional feature.